src/odemis/acq/test/orsay_milling_test.py

```diff
@@ -0,0 +1,282 @@
+# -*- coding: utf-8 -*-
+"""
+Copyright © 2020 Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+"""
+import logging
+import os
+import time
+import unittest
+from concurrent import futures
+
+from ConsoleClient.Communication.Connection import Connection
+
+import odemis
+from odemis.acq.orsay_milling import mill_rectangle
+from odemis.driver import orsay
+from odemis import model, util
+from odemis.util import testing
+from odemis.util import timeout
+
+logging.getLogger().setLevel(logging.DEBUG)
+logging.basicConfig(format="%(asctime)s  %(levelname)-7s %(module)s:%(lineno)d %(message)s")
+
+CONFIG_FIBVACUUM = {"name": "fib-vacuum", "role": "fib-vacuum"}
+CONFIG_FIBSOURCE = {"name": "fib-source", "role": "fib-source"}
+CONFIG_FIBBEAM = {"name": "fib-beam", "role": "fib-beam"}
+CONFIG_SCANNER = {"name": "scanner", "role": "scanner"}
+CONFIG_FOCUS = {"name": "focus", "role": "focus", "rng": (-2e-3, 2e-3)}
+CONFIG_DETECTOR = {"name": "detector", "role": "detector"}
+
+# Simulation:   192.168.56.101
+# Hardware:     192.168.30.101
+CONFIG_ORSAY = {"name": "Orsay", "role": "orsay", "host": "192.168.30.101",
+                "children": {"fib-vacuum": CONFIG_FIBVACUUM,
+                             "fib-source": CONFIG_FIBSOURCE,
+                             "fib-beam": CONFIG_FIBBEAM,
+                             "scanner": CONFIG_SCANNER,
+                             "focus": CONFIG_FOCUS,
+                             "detector": CONFIG_DETECTOR},
+                }
+
+NO_SERVER_MSG = "TEST_NOHW is set. No server to contact."
+CANT_FORCE_ACTUAL_MSG = "TEST_NOHW is not set to sim, cannot force data on Actual parameters of Orsay server outside " \
+                        "of simulation. "
+
+CONFIG_PATH = os.path.dirname(odemis.__file__) + "/../../install/linux/usr/share/odemis/"
+MIMAS_CONFIG = CONFIG_PATH + "sim/mimas-sim.odm.yaml"
```

Comment:
The mimas-sim.odm.yaml will not work, as it doesn't use the Orsay server. 
We have three options: 
* Rely on a back-end to be already running. If so, make it clear at the top of the file.
* Start the backend `install/linux/usr/share/odemis/hwtest/orsay.odm.yaml` when TEST_NOHW is False
* Start the backend `install/linux/usr/share/odemis/hwtest/orsay.odm.yaml` when TEST_NOHW=sim (because this file actually contains by default the IP address to the VM), and rely on the back-end to be already running if TEST_NOHW is False.

For now the Orsay simulator doesn't support the milling. So I'd go for option 1, which is the simplest.